### PR TITLE
en: update docs for getting started

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

- Update 3 docs for getting started.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/91
- Other reference link(s):

### CODE REVIEW

File: en/deploy-tidb-from-kubernetes-gke.md [hunk 1]
```diff
@@ -86,11 +86,77 @@ kubectl get nodes
...
 If you see `Ready` for all nodes, congratulations! You've setup your first Kubernetes cluster.
 
-## Deploy TiDB Operator and TiDB cluster
+## Install Helm
 
-1. Install Helm and add the Helm chart repository maintained by PingCAP. For details, refer to [Use Helm](tidb-toolkit.md#use-helm)
+[Helm](https://helm.sh/) is a package management tool for Kubernetes. Make sure your Helm version >= 2.11.0 and < 2.16.4. The installation steps are as follows:
 
-2. [Deploy TiDB Operator](deploy-tidb-operator.md#install-tidb-operator).
+1. Refer to [Helm official documentation](https://v2.helm.sh/docs/using_helm/#installing-helm) to install Helm client.
```
Comment (lilin90) [inline en/deploy-tidb-from-kubernetes-gke.md:93]:
```suggestion
1. Refer to [Helm official documentation](https://v2.helm.sh/docs/using_helm/#installing-helm) to install the Helm client.
```

File: en/deploy-tidb-from-kubernetes-gke.md [hunk 2]
```diff
@@ -86,11 +86,77 @@ kubectl get nodes
...
+
+2. Install Helm server.
```
Comment (lilin90) [inline en/deploy-tidb-from-kubernetes-gke.md:95]:
```suggestion
2. Install the Helm server.
```

File: en/deploy-tidb-from-kubernetes-gke.md [hunk 3]
```diff
@@ -86,11 +86,77 @@ kubectl get nodes
...
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    helm search pingcap -l
+    ```
+
+## Deploy TiDB Operator
+
+TiDB Operator uses [CRD (Custom Resource Definition)](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/) to extend Kubernetes. Therefore, to use TiDB Operator, you must first create the `TidbCluster` custom resource type.
```
Comment (lilin90) [inline en/deploy-tidb-from-kubernetes-gke.md:130]:
```suggestion
TiDB Operator uses [CRD (Custom Resource Definition)](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/) to extend Kubernetes. Therefore, to use TiDB Operator, you must first create the `TidbCluster` CRD.
```

File: en/deploy-tidb-from-kubernetes-gke.md [hunk 4]
```diff
@@ -86,11 +86,77 @@ kubectl get nodes
...
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/crd.yaml && \
+kubectl get crd tidbclusters.pingcap.com
+```
+
+After `TidbCluster` custom resource type is created, install TiDB Operator in your Kubernetes cluster.
```
Comment (lilin90) [inline en/deploy-tidb-from-kubernetes-gke.md:139]:
```suggestion
After `TidbCluster` CRD is created, install TiDB Operator in your Kubernetes cluster.
```

File: en/deploy-tidb-from-kubernetes-gke.md [hunk 5]
```diff
@@ -100,26 +166,60 @@ If you see `Ready` for all nodes, congratulations! You've setup your first Kuber
     kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/gke/persistent-disk.yaml
     ```
 
-4. Deploy the TiDB cluster, as in [Deploy TiDB on General Kubernetes](deploy-on-general-kubernetes.md#deploy-tidb-cluster). Set the `storageClassName` of all components to `pd-ssd`.
+## Deploy the TiDB cluster
+
+Deploy the TiDB cluster by the following steps:
```
Comment (lilin90) [inline en/deploy-tidb-from-kubernetes-gke.md:171]:
```suggestion
To deploy the TiDB cluster, perform the following steps:
```

File: en/deploy-tidb-from-kubernetes-gke.md [hunk 6]
```diff
@@ -100,26 +166,60 @@ If you see `Ready` for all nodes, congratulations! You've setup your first Kuber
...
+    {{< copyable "shell-regular" >}}
+
+    ``` shell
+    kubectl get po -n demo
+    ```
 
 ## Connect to the TiDB cluster
 
-There can be a small delay between the pod being up and running, and the service being available. You can watch list services available with:
+There can be a small delay between the pod being up and running, and the service being available. You can view the service status with the following command:
```
Comment (lilin90) [inline en/deploy-tidb-from-kubernetes-gke.md:207]:
```suggestion
There can be a small delay between the pod being up and running, and the service being available. You can view the service status using the following command:
```

File: en/deploy-tidb-from-kubernetes-kind.md [hunk 7]
```diff
@@ -108,15 +108,90 @@ First, make sure that Docker is running. Then, you can create a local Kubernetes
...
+
+Use `helm search` to search the chart provided by PingCAP:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+helm search pingcap -l
+```
+
+TiDB Operator uses [CRD (Custom Resource Definition)](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/) to extend Kubernetes. Therefore, to use TiDB Operator, you must first create the `TidbCluster` custom resource type.
```
Comment (lilin90) [inline en/deploy-tidb-from-kubernetes-kind.md:129]:
```suggestion
TiDB Operator uses [CRD (Custom Resource Definition)](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/) to extend Kubernetes. Therefore, to use TiDB Operator, you must first create the `TidbCluster` CRD.
```

File: en/deploy-tidb-from-kubernetes-kind.md [hunk 8]
```diff
@@ -108,15 +108,90 @@ First, make sure that Docker is running. Then, you can create a local Kubernetes
...
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/crd.yaml && \
+kubectl get crd tidbclusters.pingcap.com
+```
+
+After `TidbCluster` custom resource type is created, install TiDB Operator in your Kubernetes cluster:
```
Comment (lilin90) [inline en/deploy-tidb-from-kubernetes-kind.md:138]:
```suggestion
After `TidbCluster` CRD is created, install TiDB Operator in your Kubernetes cluster:
```

File: en/deploy-tidb-from-kubernetes-kind.md [hunk 9]
```diff
@@ -108,15 +108,90 @@ First, make sure that Docker is running. Then, you can create a local Kubernetes
...
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    helm install pingcap/tidb-operator --name=tidb-operator --namespace=tidb-admin --version=v1.1.0-rc.1 -f /home/tidb/tidb-operator/values-tidb-operator.yaml && \
+    kubectl get po -n tidb-admin -l app.kubernetes.io/name=tidb-operator
+    ```
+
+## Step 3: Deploy the TiDB cluster
+
+Deploy the TiDB cluster by the following steps:
```
Comment (lilin90) [inline en/deploy-tidb-from-kubernetes-kind.md:162]:
```suggestion
To deploy the TiDB cluster, perform the following steps:
```

File: en/deploy-tidb-from-kubernetes-minikube.md [hunk 10]
```diff
@@ -62,30 +62,132 @@ After kubectl is installed, test your minikube Kubernetes cluster:
...
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    helm search pingcap -l
+    ```
+
+## Deploy TiDB Operator
+
+TiDB Operator uses [CRD (Custom Resource Definition)](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/) to extend Kubernetes. Therefore, to use TiDB Operator, you must first create the `TidbCluster` custom resource type.
```
Comment (lilin90) [inline en/deploy-tidb-from-kubernetes-minikube.md:108]:
```suggestion
TiDB Operator uses [CRD (Custom Resource Definition)](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/) to extend Kubernetes. Therefore, to use TiDB Operator, you must first create the `TidbCluster` CRD.
```

File: en/deploy-tidb-from-kubernetes-minikube.md [hunk 11]
```diff
@@ -62,30 +62,132 @@ After kubectl is installed, test your minikube Kubernetes cluster:
...
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/crd.yaml && \
+kubectl get crd tidbclusters.pingcap.com
+```
+
+After `TidbCluster` custom resource type is created, install TiDB Operator in your Kubernetes cluster.
```
Comment (lilin90) [inline en/deploy-tidb-from-kubernetes-minikube.md:117]:
```suggestion
After `TidbCluster` CRD is created, install TiDB Operator in your Kubernetes cluster.
```

File: en/deploy-tidb-from-kubernetes-minikube.md [hunk 12]
```diff
@@ -62,30 +62,132 @@ After kubectl is installed, test your minikube Kubernetes cluster:
...
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    helm install pingcap/tidb-operator --name=tidb-operator --namespace=tidb-admin --version=v1.1.0-rc.1 -f /home/tidb/tidb-operator/values-tidb-operator.yaml && \
+    kubectl get po -n tidb-admin -l app.kubernetes.io/name=tidb-operator
+    ```
+
+## Deploy the TiDB cluster
+
+Deploy the TiDB cluster by the following steps:
```
Comment (lilin90) [inline en/deploy-tidb-from-kubernetes-minikube.md:141]:
```suggestion
To deploy the TiDB cluster, perform the following steps:
```

**Comment by ran-huang** [comment]
@lilin90 Comments addressed. PTAL~

**Pull request merged**